models/gender_model.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import keras as ke
+from keras import backend as K
+from keras.models import Sequential, Model
+from keras.layers import Activation, Input, Flatten, Dense
+from keras.layers.core import Dense, Flatten
+from keras.optimizers import Adam
+from keras.metrics import categorical_crossentropy
+from keras.preprocessing.image import ImageDataGenerator
+from keras.layers.normalization import BatchNormalization
+from keras.layers.convolutional import *
+from matplotlib import pyplot as plt
+from sklearn.metrics import confusion_matrix
+import itertools
+import matplotlib.pyplot as plt
+
+
+# Absolut paths have to get adjusted
```

Comment:
https://stackoverflow.com/questions/44772007/when-to-use-absolute-path-vs-relative-path-in-python